packages/server/src/api/controllers/table/utils.ts

```diff
@@ -91,26 +89,6 @@ export async function checkForColumnUpdates(
     await checkForViewUpdates(updatedTable, deletedColumns, columnRename)
   }
 
-  const changedAttachmentSubtypeColumns = Object.values(
```

Comment:
Unused?

Reply:
Yup, we got rid of this single subtype in favour of the single attachment column. This has not been used in prod, so safe to remove